EIPS/eip-5850.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: 5850
+title: Complex Numbers stored in Bytes32 types
+description: Store real and imaginary parts of complex numbers in the least significant 16 bytes and most significant 16 bytes respectively of a bytes32 type.
+author: Paul Edge (@genkifs)
+discussions-to: https://ethereum-magicians.org/t/eip-5850-store-real-and-imaginary-parts-of-complex-numbers-in-the-least-significant-and-most-significant-16-bytes-respectively-of-a-bytes32-type/11532
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-29
+---
+ 
+## Abstract
+
+This EIP proposes a natural way for complex numbers to be stored in and retrieved from the bytes32 data-type.  It splits the storage space exactly in half and, most importantly, assigns the real number part to the least significant 16 bytes and the imaginary number part to the most significant 16 bytes. 
+
+## Motivation
+
```

Comment:
I still don't see a particularly compelling use case that involves manipulating complex numbers on the blockchain.

Reply:
I've added these use cases to the EIP [Fourier Transforms](https://en.wikipedia.org/wiki/Fourier_transform), [Characteristic functions](https://en.wikipedia.org/wiki/Characteristic_function_(probability_theory\)), [AC Circuits](http://www.mathforengineers.com/AC-circuits/complex-numbers-in-AC-circuits.html), [Navier-Stokes equations](https://en.wikipedia.org/wiki/Navier%E2%80%93Stokes_equations). 

Note that this EIP doesn't discuss manipulation of complex numbers, just storage.